test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Image.html

```diff
@@ -9,7 +9,7 @@
 
     <h2>Image Tag Helper Test</h2>
     <!-- Plain image tag -->
-    <img src="/images/red.png" alt="Red block" title="&lt;the title>">
+    <img src="/images/red.png" alt="Red block" title="&lt;the title>" />
```

Comment:
The `.cshtml` file was incorrect whether or not the `UrlResolutionTagHelper` exists. And the `UrlResolutionTagHelper` doesn't care whether the element is void. So, it's a separate fix that's worth calling out in the commit description.